Makefile

```
@@ -188,12 +192,11 @@ help:  # Display this help
 
 ##@ generate:
 
-ALL_GENERATE_MODULES = core kubeadm-bootstrap kubeadm-control-plane
+ALL_GENERATE_MODULES = core kubeadm-bootstrap kubeadm-control-plane capd
```

```suggestion
ALL_GENERATE_MODULES = core kubeadm-bootstrap kubeadm-control-plane
```
(as we now dropped the target)

---

I think you meant `ALL_VERIFY_CHECKS` instead